loader/loader.go

```diff
@@ -837,7 +846,7 @@ func (l *Loader) restoreSchema(conn *Conn, sqlFile, schema string) error {
 	err := l.restoreStructure(conn, sqlFile, schema, "")
 	if err != nil {
 		if isErrDBExists(err) {
-			log.Infof("[loader][database already exists, skip]%s", sqlFile)
+			l.tctx.L().Info("database already exists, skip it", zap.String("sql", sqlFile))
```

Comment:
Can we unify the key for file? now, `sql`, `file` and `data file` exist.
or, are you trying to differ from _schema file_, _table file_ and _data file_?

Reply:
In addition to these two, the other are the variable they should have.